libsolidity/inlineasm/AsmAnalysis.cpp

```diff
@@ -49,6 +49,22 @@ bool AsmAnalyzer::analyze(Block const& _block)
 	if (!(ScopeFiller(m_info, m_errorReporter))(_block))
 		return false;
 
+	/// Populate top level block with builtin functions
+	if (m_julia)
+	{
+		Scope& rootScope = scope(&_block);
+		rootScope.registerFunction("abort", {}, {});
```

Comment:
I had the same code, but for some reason I got an error when providing the list as constructor parameters to the vector.